.github/workflows/labeler.yaml

```diff
@@ -9,8 +9,9 @@ concurrency:
 jobs:
   triage:
     permissions:
-      contents: read
+      contents: write
       pull-requests: write
+      issues: write
```

Comment:
Can we revert these, please? Now that we know labels on the repo level need to be created first.